custom_components/unifiprotect/config_flow.py

```diff
@@ -197,6 +199,10 @@ async def async_step_init(self, user_input=None):
                         CONF_DISABLE_RTSP,
                         default=self.config_entry.options.get(CONF_DISABLE_RTSP, False),
                     ): bool,
+                    vol.Optional(
+                        CONF_ALL_UPDATES,
+                        default=self.config_entry.options.get(CONF_ALL_UPDATES, False),
+                    ): bool,
```

Comment:
Is there a use case for users turning this one who are not devs?

Reply:
Not yet. But I planned on adding more diagnostic sensors for things like WiFi strength, etc and those would likely require that enabled.

Reply:
Actually, one sensor, _might_ need it now already. I the UP Sense already has a Bluetooth strength sensor.